drift/info_parser.py

```diff
@@ -187,6 +184,12 @@ def _group_comparisons(comparisons):
         else:
             grouped_comparisons.append(comparison)
 
+    # sorting comparisons if exists
+    for comparison_name in grouped_comparisons_with_dotnotation:
+        group = grouped_comparisons_with_dotnotation.get(comparison_name)
+        group["comparisons"] = sorted(
```

Comment:
Can it be done without a for loop (or at least without the get by key) and sorted inplace?

Reply:
I believe not, since we're sorting a child item, we need to pick up the object and sort it.

Reply:
This for loop
```
for comparison_name in grouped_comparisons_with_dotnotation:
```
gives keys.

Can it be changed to get `.values()` only and sort inplace? The similar way as you recently did? List has a sort method. `sorted` is a generic builtin and creates a new list. We should reduce the number of objects being created.

To make it without a for-loop, there are comperhensions which can be quicker with the cost of readability.